app/src/main/java/com/x8bit/bitwarden/data/autofill/accessibility/manager/AccessibilityActivityManagerImpl.kt

```diff
@@ -0,0 +1,50 @@
+package com.x8bit.bitwarden.data.autofill.accessibility.manager
+
+import android.content.Context
+import android.provider.Settings
+import androidx.lifecycle.LifecycleCoroutineScope
+import com.x8bit.bitwarden.data.platform.manager.AppForegroundManager
+import kotlinx.coroutines.flow.launchIn
+import kotlinx.coroutines.flow.onEach
+
+/**
+ * THe accessibility service name as it is known in the android manifest.
+ */
+private const val ACCESSIBILITY_SERVICE_NAME =
```

Comment:
❓  and the answer may just be, this is how we want to treat string constants. Could we have this as a public constant in `BitwardenAppComponentFactory` or maybe have some `object BitwardenAppComponents { const val accessibiltityServiceName = "com.x8bit.yada.yada.yada" `

Reply:
That seems like a good idea, I'd like to make a separate PR where we can do that in a consistent way for all of those legacy app component names.

Reply:
Sounds good!